src/wallet/rpcwallet.cpp

```diff
@@ -2470,9 +2556,6 @@ UniValue listunspent(const UniValue& params, bool fHelp)
         const CScript& scriptPubKey = out.tx->vout[out.i].scriptPubKey;
         bool fValidAddress = ExtractDestination(scriptPubKey, address);
 
-        if (destinations.size() && (!fValidAddress || !destinations.count(address)))
-            continue;
-
```

Comment:
I don't remember the reason for this change. Is it that `AvailableCoins` now performs this filtering?

Reply:
Yeah, sorry, I think I prematurely resolved that conversation. It’s here: https://github.com/zcash/zcash/pull/6122/files/5ea600a102d6e2e4a062570981d00662ccb41b0e#r1030770197

Reply:
Since the context of the comment changed a bit because of commits, here’s that change: https://github.com/zcash/zcash/pull/6122/commits/583fbb22180aad59878eb8135a43e51bba09098